tests/src/Kernel/EventIntegrationTest.php

```diff
@@ -157,4 +157,32 @@ public function testSystemLoggerEvent() {
     $this->assertRulesLogEntryExists('action called');
   }
 
+  /**
+   * Test that Drupal initializing triggers the Rules logger listener.
+   */
+  public function testInitEvent() {
+    $rule = $this->expressionManager->createRule();
+    $rule->addCondition('rules_test_true');
+    $rule->addAction('rules_test_log');
+
+    $config_entity = $this->storage->create([
+      'id' => 'test_rule',
+      'expression_id' => 'rules_rule',
+      'event' => 'rules_system_logger_event',# @todo which event should we use?
```

Comment:
Please read the links I sent you, they explain that the init event does not exist and the comment on the link goes to a sandbox showing code on how to mimic the init event in 8.